proposals/005-skip-approval-process/README.md

```diff
@@ -0,0 +1,51 @@
+# SKIP 005 - SKIP Approval Process
+
+Summary: A process for approving future SKIPs.
+
+Owner: Caleb Schoepp <caleb.schoepp@fermyon.com>
+
+Impacted Projects:
+
+- [x] spin-operator
+- [x] `spin kube` plugin
+- [x] runtime-class-manager
+- [x] containerd-shim-spin
+- [x] Governance
+- [ ] Creates a new project
+
+Created: 30/10/2024
+
+Updated: n/a
+
+## Background
+
+SKIPs were designed as a lightweight way to have discussions about changes that are more complex or affect multiple projects. They have worked well for this purpose, but we never codified how we should reach agreement around approving a SKIP. This proposal seeks to change that.
+
+## Proposal
+
+The following conditions must be met for a SKIP to be considered approved:
+
+- The SKIP has been open for comment for at least two weeks.
```

Comment:
Do we need "and any objecting comments are resolved"? For example, this skip i may has been open for 2 weeks but i dont think it should be merged: https://github.com/spinkube/skips/pull/14

Reply:
I think that is a good addition. I'll add it.